projects/gnomad/src/client/DownloadsPage.js

```diff
@@ -286,17 +305,18 @@ export default () => (
               >
                 gs://gnomad-public/release/2.1/coverage/exomes/gnomad.exomes.r2.1.coverage.ht
               </ExternalLink>
+              <br />
             </li>
           </ul>
         </DownloadSection>
         <DownloadSection>
           <h3>VCFs</h3>
           <ul>
             <li>
-              <ExternalLink href="https://storage.googleapis.com/gnomad-public/release/2.1/vcf/exomes/gnomad.exomes.r2.1.sites.vcf.bgz">
-                All chromosomes (59.23 GiB){' '}
+              <ExternalLink href="https://storage.googleapis.com/gnomad-public/release/2.1.1/vcf/exomes/gnomad.exomes.r2.1.sites.vcf.bgz">
```

Comment:
These two links are incorrect. They have the "2.1.1" folder but the filename contains "2.1.sites".